pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -58,14 +58,22 @@ kubectl rollout pause deployment/nginx`
 func NewCmdRolloutPause(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	opts := &PauseConfig{}
 
+	allErrs := []error{}
```

Comment:
Could you please provide some more context by what you mean here?

I collect the errors in `NewCmd<CmdName>` because if `Complete<CmdName>` fails then so does `Run<CmdName>`, which would not let the processing of the entire directory continue upon the first error encountered.